src/pdn/test/power_switch.api.tcl

```diff
@@ -0,0 +1,52 @@
+source "helpers.tcl"
+
+read_lef ../../../test/sky130hd/sky130hd.tlef 
+read_lef ../../../test/sky130hd/sky130_fd_sc_hd_merged.lef 
+
+read_lef sky130hd/power_switch.lef 
```

Comment:
please move to sky130_power_switch/ (keep test files together)

Reply:
Done